modules/rgbd/src/tsdf_functions.hpp

```diff
@@ -43,6 +43,47 @@ void integrateVolumeUnit(
     InputArray _depth, float depthFactor, const cv::Matx44f& cameraPose,
     const cv::kinfu::Intr& intrinsics, InputArray _pixNorms, InputArray _volume);
 
+
+struct Volume_NODE
+{
+    Vec4i idx = Vec4i(-2147483647);
+    int32_t row   = -1;
+    int32_t nextVolumeRow = -1;
+    int32_t isActive = 0;
+    int32_t lastVisibleIndex = -1;
+    //int32_t tmp;
+};
+
+size_t calc_hash(Vec4i x);
+
+class VolumesTable
+{
+public:
+    int hash_divisor = 32768;
```

Comment:
If these ones are constants, this should be written explicitly

Reply:
done